tests/integration/HttpClientTest.php

```diff
@@ -60,4 +60,19 @@ public function testHttpClientWithoutAuthenticationException()
         $client = new HttpClient(Client::ENDPOINT);
         $client->performHttpRequest('foo', 'bar');
     }
+
+    /**
+     * Test that it allows additional CURL options 
+     */
+     public function testHttpClientWithHttpOptions() 
```

Comment:
Please check the indentation of this function: indentation should be 4 spaces, but function has 5